Various technical notes/transition_infos.json

```diff
@@ -660,13 +654,7 @@
       "area_id": "0x4B08BBEB",
       "area_name": "Scorpion Temple (Harry)",
       "default_entrance": "0x9A0C8DF8",
-      "exits": [
-        {
-          "area_id": "0x9A0C8DF8",
-          "area_name": "Eyes of Doom",
-          "requires": []
-        }
-      ]
+      "exits": []
```

Comment:
Why remove this?

```suggestion
      "exits": [
        {
          "area_id": "0x9A0C8DF8",
          "area_name": "Eyes of Doom",
          "requires": []
        }
      ]
```

Reply:
As mentioned in both the commit titles as well as the utils.py file that writes it to the Spoiler Log:
**Scorpion Temple (Harry)** is temporarily removed from the randomization proccess. An essential part of that removal is that this exit is not included in the .json file.

Reply:
I mark this as resolved because it's a duplicate of another conversation.